pbs_light.go

```diff
@@ -533,7 +535,13 @@ func sortBidsAddKeywordsMobile(bids pbs.PBSBidSlice, pbs_req *pbs.PBSRequest, pr
 					pbs_kvs[hbCreativeLoadMethodConstantKey] = hbCreativeLoadMethodHTML
 				}
 			}
-			bid.AdServerTargeting = pbs_kvs
+			if bid.AdServerTargeting == nil {
```

Comment:
Can you leave a `// Fixes #288` comment here?

If it was subtle enough to become a bug once, it's likely to become one again during some future refactoring. That should help point someone to the github issue so they don't re-introduce it.

A regression test would also be nice... although in this case it's not super important, because this is in the legacy code and will be obsoleted fairly soon.